### DISCUSSION TITLE
Can it open zip having pdf files?

### DISCUSSION THREAD

**Discussion by Dowbyun** [sumatrapdfreader/sumatrapdf#2914] [General]
If a book comprises in each separaed pdf page, can it open after compressing each page whole together in one zip file.

**Comment by GitHubRulesOK** [comment]
zip archive files should not contain PDF archive for reading there are problems reading PDF files that are not freely available to manipulate on a hard drive or similar filesystem and inside a zip they cannot be self decompressed to read some encoded features or to allow comments printing etc. 
Any reader of files in a zip such as comic books, word docs or epub needs to expand / extract from those zips as SumatraPDF can, but nested one inside another is not normal.
